.pre-commit-config.yaml

```diff
@@ -3,23 +3,19 @@ repos:
     rev: 19.10b0
     hooks:
     -   id: black
-        language_version: python3
```

Comment:
This is already specified here: https://github.com/psf/black/blob/master/.pre-commit-hooks.yaml